openlibrary/templates/lists/dropper_lists.html

```diff
@@ -1,7 +1,14 @@
-$def with (lists)
+$def with (lists, legacy_rendering=True)
 
 $for list in lists:
-  $if not list['active']:
-    <p class="list">
-      <a href="$list.key" class="add-to-list dropper__close" data-list-cover-url="$list.cover_url" data-list-key="$list.key">$list.name</a>
+  $if legacy_rendering:
+    $if not list['active']:
+      <p class="list">
+        <a href="$list.key" class="add-to-list dropper__close" data-list-cover-url="$list.cover_url" data-list-key="$list.key">$list.name</a>
+      </p>
+  $else:
+    $ list_active = "list--active" if list['active'] else ""
+    <p class="list $list_active">
+      <span class="check">✔️</span>
+      <a href="$list.key" class="modify-list dropper__close" data-list-cover-url="$list.cover_url" data-list-key="$list.key" data-list-items="$(json_encode(list.list_items))">$list.name</a>
```

Comment:
```suggestion
      <a href="$list.key" class="modify-list" data-list-cover-url="$list.cover_url" data-list-key="$list.key" data-list-items="$(json_encode(list.list_items))">$list.name</a>
```
Prevent dropper from closing when the list affordance is clicked.